snapshots.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what's this file?

Reply:
Well good question. I have copied the example from the window spy folder and adjusted for my use case. If those files are irrelevant, they should be removed from that example, too. What do you think?

Reply:
yeah remove it from this file and from window recipe

Reply:
I have deleted this file. No idea how this file was created. I will open an issue to cleanup the existing examples. Maybe there are more files that are not necessary.

Reply:
https://github.com/cypress-io/cypress-example-recipes/issues/348